### ISSUE TITLE
poetry run on Windows [WinError 193] %1

### ISSUE THREAD

**Issue by yaseralnajjar** [adamghill/django-unicorn#93]
When running `poetry run` on Windows, I ran into this issue:

```
λ poetry run example/manage.py migrate

  OSError

  [WinError 193] %1 is not a valid Win32 application

  at c:\users\kasse\appdata\local\programs\python\python39\lib\subprocess.py:1416 in _execute_child
      1412│             sys.audit("subprocess.Popen", executable, args, cwd, env)
      1413│
      1414│             # Start the process
      1415│             try:
    → 1416│                 hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
      1417│                                          # no special security
      1418│                                          None, None,
      1419│                                          int(not close_fds),
      1420│                                          creationflags,
```

**Comment by yaseralnajjar** [comment]
Fixed here: https://github.com/adamghill/django-unicorn/pull/94

**Comment by adamghill** [comment]
Closing this issue since I merged #94.
